### ISSUE TITLE
[Profile form] Hitting personal email address error needs to logout user

### ISSUE THREAD

**Issue by meissadia** [cfpb/sbl-frontend#196]
We currently [do not display a Logout button on the `/profile-form` page](https://github.com/cfpb/sbl-frontend/blob/main/src/App.tsx#L89).  This means that if a user accidentally signs in with a personal email address, and would like to switch to their business account, they have no way of invalidating the initial login.
 
I propose that we do show the account actions in the page header to allow users flexibility. 

![Image](https://github.com/cfpb/sbl-frontend/assets/2592907/8f598be6-cc9b-41de-ae0e-10cf2633ab25)

**Comment by billhimmelsbach** [comment]
Talked about this with Natalia last week a bit. Not having the log out or user profile link on that unauthenticated landing page was intentionally not part of the initial designs. That's why there is an exception for it. I think we've all decided it's good to have though now?

**Comment by natalia-fitzgerald** [comment]
@meissadia 
A user that lands here should click on the link in the notification and go to Login.gov to switch the account that they are using to log in. If a user sees log out here and clicks it they are only logging out of Keycloak, is that correct? Wouldn't this user assume that they are logging out of Login.gov and not Keycloak (a system that is invisible to them)?  A user cannot access the platform with a personal email address so wouldn't giving them the option to logout or go to their profile suggest that they are in?

**Comment by natalia-fitzgerald** [comment]
@billhimmelsbach 
I don't think so. I am halfway through commenting in the Figma page regarding the unauthenticated header space. I don't want this discussion to get lost here so I'll move it to there.

**Comment by billhimmelsbach** [comment]
@natalia-fitzgerald It's a bad user interaction for sure, but the current behavior is also... bad. I talked about this same behavior with Kenneth at the RegTech All Hands regarding the HMDA platform, and when users log out of there they also don't get logged out of login.gov because their team is inflexible about that process. We could redirect them to Login.gov AFTER logging them out of Keycloak, but then we might also be sending them to a screen with no clear way back to our app. I also confirmed that we don't have a way of knowing if someone is still logged into login.gov currently, so we might be sending them to a screen that shows them not being logged in already.

**Comment by billhimmelsbach** [comment]
We're moving this discussion over to our design board. 👍

**Comment by natalia-fitzgerald** [comment]
@meissadia 
In the DSR/SBL Front end dev meeting we discussed exploring having the system log the user out of Keycloak automatically when they land on this page. @billhimmelsbach - Can you add any additional context?

**Comment by billhimmelsbach** [comment]
Yep! From my notes + some extra context:

   * We decided not to include the navbar on the error page because a user doesn't need to know that they are logged into keycloak generally, specifically when they're logged in a way that prevents them from going any further
   * Users should just be logged out of keycloak in the background when they reach this error view since that's what all users who reach this page will all need to do anyway (we'll have to investigate what's possible there)
   * @natalia-fitzgerald will add a comment on Figma explaining the new log out behavior and that the language around adding a professional email to your existing login.gov account will stay the same and tag everyone (Alex, Bill)
